gpii/node_modules/solutionsRegistry/src/schemas/solution-schema.json5

```diff
@@ -49,7 +49,14 @@
                         "type": "object",
                         "additionalProperties": {
                             "type": "object",
-                            "additionalProperties": false,
+                            "additionalProperties": {
```

Comment:
I could swear I already pointed this issue out to JJ.  The nesting of `additionalProperties` here is too deep and probably wrong-headed.  The intent is probably to say that any supported setting can have an optional label, and that it has to be a string.  The actual implementation says that any supported setting can contain an object that contains a label.  The `label` block should just be in `properties` instead.

Reply:
I've moved this to being a regular property as it should be next to schema, which matches what I'm seeing in win32.json.